src/utils/webrtc/webrtc.js

```diff
@@ -552,10 +552,17 @@ export default function initWebRTC(signaling, _callParticipantCollection, _local
 		webrtc.leaveCall()
 	})
 
-	webrtc.startMedia = function(token) {
+	webrtc.startMedia = function(token, flags) {
 		startedWithMedia = undefined
 
-		webrtc.joinCall(token)
+		// If no flags are provided try to enable both audio and video.
+		// Otherwise, try to enable only that allowed by the flags.
```

Comment:
```suggestion
		// Otherwise, try to enable only those allowed by the flags.
```

Reply:
Is not `that` correct there? Maybe I am just doing a direct translation from Spanish, but sounds good to me :thinking: